src/views/hoc/hoc.scss

```diff
@@ -8,123 +8,117 @@ $base-bg: $ui-white;
 }
 
 .hoc {
```

Comment:
Oh – I thought that part of our customization from Trello would be to keep this nesting for the scoping... I'll take a look at how this might be done without causing conflicts with the way other views/components currently work.